docs/documentation/guides/nextjs-vercel.mdx

```diff
@@ -241,7 +241,7 @@ At this stage, you know how to use the Infisical-Vercel integration to sync prod
         Check out the [security guide](/security/overview).
     </Accordion>
     <Accordion title="Is there way to retain end-to-end encryption for syncing production secrets to Vercel?">
-        Yes. You can also use the Infisical [Node SDK](https://github.com/Infisical/infisical-node) to fetch secrets back to your Next.js app
+        Yes. You can also use the Infisical [Node SDK](https://infisical.com/docs/sdks/languages/node) to fetch secrets back to your Next.js app
```

Comment:
New SDKs do not retain end-to-end encryption. We are moving away from e2ee to make it easier for customers to interact with api

Reply:
Maybe you can point to old sdks that do but say that this will soon be deprecated